src/data/translation.ts

```diff
@@ -17,10 +17,22 @@ export enum TimeFormat {
   twenty_four = "24",
 }
 
+export enum FirstWeekday {
+  language = "language",
+  monday = "monday",
+  tuesday = "tuesday",
+  wednesday = "wednesday",
+  thursday = "thursday",
+  friday = "friday",
+  saturday = "saturday",
+  sunday = "sunday",
```

Comment:
Should we make these numeric values, so we can use them as index directly?

Reply:
There is `language` option. So I preferred to keep all in string for consistency.